content/en/cosign/verify.md

```diff
@@ -1,16 +1,36 @@
 ---
-title: "Verifying Claims"
+title: "Verifying"
 category: "Cosign"
 position: 106
 ---
 
 The general verification format with the `cosign verify` command is as follows.
 
 ```shell
-$ cosign verify --key <key path>|<key url>|<kms uri> <image uri> 
+$ cosign verify [--key <key path>|<key url>|<kms uri>] <image uri> 
 ```
+## Keyless verification using OpenID Connect
 
-We'll use `user/demo` as our example image in the following commands. 
+We'll use `user/demo` as our example image in the following commands and keyless signing where appropriate.
+
+To verify a container image keylessly, use the following command:
```

Comment:
Instead of **`keylessly`**, we can use → `For keyless verification of a container image, use the following command:`

Reply:
@ltagliaferri what's your thought on this? 🤔

Reply:
Would preer "For identity-based verification of a container image" unless we've defined "keyless"